lib/devise_two_factorable/models/two_factorable.rb

```diff
@@ -118,6 +124,10 @@ def validate_otp_recovery_token(token)
 
     def validate_otp_token_with_drift(token)
       # should be centered around saved drift
+      if self.last_successful_otp_at && valid_drift_window?
```

Comment:
I'm not certain this is right... If a user doesn't have any successful otp's ever it fails?

Reply:
`self.last_successful_otp_at &&` could also be stuffed into the responsible method and this can be a nice easy guard -- if it's actually needed.

Reply:
Ah I see... See this: https://github.com/rietta-inc/devise-2fa/pull/14/files#r285274603